src/lib/remote-place-cache.ts

```diff
@@ -47,23 +70,35 @@ export default class RemotePlaceCache {
   public static add(place: Place, chtApi: ChtApi): void {
     const { domain } = chtApi.chtSession.authInfo;
     const placeType = place.type.name;
-
-    const places = this.cache[domain]?.[placeType];
-    // if there is no cache existing, discard the value
-    // it will be fetched if needed when the cache is built
-    if (places) {
+    const cacheKey = this.getCacheKey(domain, placeType);
+    
+    const places = this.getCache().get<RemotePlace[]>(cacheKey) || [];
+    
+    if (places.length === 0) {
```

Comment:
why do you only add if places is empty?  previously, we only added if not empty...